doc/source/contributing.rst

```diff
@@ -32,10 +32,28 @@ Where to start?
 All contributions, bug reports, bug fixes, documentation improvements,
 enhancements, and ideas are welcome.
 
-If you are brand new to MDAnalysis or open-source development, we recommend going
-through the guides for :ref:`contributing to the main codebase <working-with-mdanalysis-code>` or the :ref:`user guide <working-with-user-guide>`. If you are new to Git and version control, have a look at
+If you are new to Git and version control, have a look at
 :ref:`version-control`.
 
+If you are looking to contribute a code or documentation fix, or core feature to MDAnalysis,
+and you are brand new to MDAnalysis or open-source development -- we recommend going
+through the guides for :ref:`contributing to the main codebase <working-with-mdanalysis-code>`
+or the :ref:`user guide <working-with-user-guide>`. 
+
+If you are looking to contribute your own project or new addition to MDAnalysis,
+we encourage you to consider doing so by creating an
```

Comment:
We probably shouldn't block off contributing to the core library completely - here I would say something like "open an issue or consider creating an mdakit"

Reply:
Thanks -- changed!